pkg/api/server/register_kube.go

```diff
@@ -58,6 +58,11 @@ func (s *APIServer) registerKubeHandlers(r *mux.Router) error {
 	//    type: boolean
 	//    default: false
 	//    description: Clean up all objects created when a SIGTERM is received or pods exit.
+	//  - in: query
+	//    name: useLongAnnotations
```

Comment:
Here I'd prefer calling it `notrunc`.  Naming is very subjective, so I'd prefer others to weigh in as well.

Reply:
Same for other user-facing APIs (e.g., bindings).

Reply:
Ah yes, I agree. I didn't realize I did that